_episodes/01-quickstart.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Quickstart guide"
+teaching: 1
+exercises: 10
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I activate and test the ESMValTool environment?"
+- "How do I configure ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Understand the purpose of the quickstart guide"
+- "Activate and test the ESMValTool environment"
+- "Configure ESMValTool"
+- "Run a recipe"
+
+keypoints:
+
+- "The purpose of the quickstart guide is to enable a user of ESMValTool
+  to run ESMValTool as quickly as possible"
+- "Use the `module load` command to activate the ESMValTool environment 
+  (see the installation guide for more details) and use `esmvaltool -- --help` to 
+  test the ESMValTool environment"
+- "Use `esmvaltool config get_config_user` to create the ESMValTool user 
+  configuration file"
+- "Use `esmvaltool run <recipe>.yml` to run a recipe"
+---
+> ## Compatibility
+>
+> This quickstart guide is compatible with ESMValTool v2.6.0.
+ {: .callout}
+
```

Comment:
I think it would be simpler not to have the compatibility note for this episode because the instructions work with any v2+ version and that would be a way to keep the quickstart guide general. Also, it would not be necessary to update this episode after each ESMValTool release. The situation is different for other episodes where instructions and log messages are version dependent.

Reply:
Done 8ac17f7.